app/views/ReportView/components/GenomicSummary/index.tsx

```diff
@@ -197,6 +197,7 @@ const GenomicSummary = ({
           setVariants(sorted);
           setVariantCounts(counts);
           if (loadedDispatch) {
+            // TODO
```

Comment:
yes my intellisense is throwing 'type {} has no call signature' for loadedDispatch. Maybe this is working as intended already and it's just intelli-nonsense but I flagged just in case